app/assets/stylesheets/profile.scss

```diff
@@ -31,6 +31,10 @@ a:hover {
   margin-top: 5%;
   background-color: #FF851B;
   color: white;
+  &:hover {
+    background-color: #FF851B;
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.